tests/cross/src/main/scala/tests/BaseCompletionSuite.scala

```diff
@@ -54,14 +54,36 @@ abstract class BaseCompletionSuite extends BasePCSuite {
     )
   }
 
+  /**
+   * Check completions using `fn` returned at @@ cursor position indicated in the `original` string.
+   *
+   * @param name name of the test
+   * @param original snippet to test with `@@` indicating the cursor position
+   * @param fn function used to run assertions on completion items
+   */
   def checkItems(
       name: TestOptions,
       original: String,
-      fn: Seq[CompletionItem] => Unit
+      fn: Seq[CompletionItem] => Boolean
```

Comment:
Checked the method here, since it was misleading and people might have assumed that it runs assert on fn returned code..